jmx-scraper/src/main/java/io/opentelemetry/contrib/jmxscraper/JmxConnectorBuilder.java

```diff
@@ -138,6 +138,7 @@ private Map<String, Object> buildEnv() {
   @SuppressWarnings("BanJNDI")
   private static JMXConnector doConnect(JMXServiceURL url, Map<String, Object> env)
       throws IOException {
+    logger.info("Connecting to " + url);
```

Comment:
maybe better to be debug, i'm not sure about jmx-scraper but we are generally pretty light on info logs (outside of tests)

```suggestion
    logger.debug("Connecting to {}", url);
```

Reply:
So this happens only once on application startup and the url is not printed anywhere else. I think it may be useful  for connection issue investigations because default Java error message for JMX is quite cryptic (like "Unable to connect Unable to connect to :0" for invalid host). Would you agree to leave it as is?

Reply:
> Would you agree to leave it as is?

👍